packages/blockchain-extension/extension/commands/exportWalletCommand.ts

```diff
@@ -62,16 +62,42 @@ export async function exportWallet(walletTreeItem?: WalletTreeItem): Promise<voi
         return;
     }
 
-    // Copy the wallet to the chosen location
+    const walletName: string = exportedWalletRegistryEntry.displayName ? exportedWalletRegistryEntry.displayName : exportedWalletRegistryEntry.name;
+
     try {
-        await fs.copy(exportedWalletRegistryEntry.walletPath, walletUri.fsPath, { overwrite: true });
+        // create parent directory
+        await fs.ensureDir(walletUri.fsPath);
+        const walletContents: string[] = await fs.readdir(exportedWalletRegistryEntry.walletPath);
+
+        if (walletContents.length) {
+            for (const walletItem of walletContents) {
+                const walletItemPath: string = path.join(exportedWalletRegistryEntry.walletPath, walletItem);
+                // looking for directories that will contain identity information
+                if ((await fs.lstat(walletItemPath)).isDirectory()) {
```

Comment:
this won't work for v2, wallets have a new format. So in the wallet dir you have files that are called `<identityName>.id`